tempesta_fw/t/functional/systemtap/resp_alloc_err.stp

```diff
@@ -0,0 +1,6 @@
+probe module("tempesta_fw").function("tfw_http_msg_alloc_err_resp").return
+{
+	if ($return != 0) {
+		$return = 0
+	}
```

Comment:
Make sense. There is nothing special about preventing memory leakage: `tfw_pool_destroy()` is quite straightforward and short function and you can just reimplement it in the hook, basically it just frees the allocated pages.

Reply:
Yes, I've missed this approach. Corrected.